src/main/java/edu/usf/cutr/grha/model/ChicagoTncData.kt

```diff
@@ -38,4 +39,11 @@ data class ChicagoTncData
     @Parsed val pickupCentroidLongitude: Double = 0.0,
     @Parsed val dropoffCentroidLatitude: Double = 0.0,
     @Parsed val dropoffCentroidLongitude: Double = 0.0,
+    @Parsed var totalTravelTime: Int? = 0,
```

Comment:
I believe we want these metrics for the top 3 trip plan results, IIRC? So for this to output cleanly to CSV I think we'd need to label these "...1", and then have "...2", and "...3" for all the fields.

Reply:
Ahh - I get it. What if there are less than 3 ways to reach a destination? I guess they'll hold the default values then.

Reply:
Quick clarification, should the top 3 be as-is or should we sort by travel time?

Reply:
>Quick clarification, should the top 3 be as-is or should we sort by travel time?

I would leave them sorted as-returned by OTP, because they are ordered by OTP's preference.

We should probably include somehow the priority preference used in the OTP requests, maybe as new model field? QUICK, SAFEST, etc. This obviously impacts the sorting.

Reply:
Right, a common model field for all the 3 trip plan results that shows what the `OptimizeType` was.